src/main/resources/application.yml

```diff
@@ -43,8 +43,11 @@ messaging:
             password: guest
             virtualHost: "/"
             camelRoute:
-                destinationName: engineEvents
+                destinationName: acsCloudConnectorUpdateMetadata
                 toRoute: rabbitmq:${messaging.to.rabbitmq.camelRoute.destinationName}?connectionFactory=#rabbitmqConnectionFactory&exchangeType=topic&autoDelete=false
+                routes:
```

Comment:
Were we considering naming this `endpoints` to avoid confusion with routes?

Reply:
There is no need for **_routes_** here. **_CamelRoute_** should be updated to accept a list of _destinationName_ and _toRoute/endpoint_.